Public/Src/Utilities/Native/Processes/Windows/ProcessUtilities.Win.Container.cs

```diff
@@ -217,8 +232,9 @@ private static void ConfigureWciFilter(IntPtr hJob, IReadOnlyCollection<Expanded
             
         }
 
-        private static void ConfigureBindFilter(IntPtr hJob, IReadOnlyCollection<ExpandedAbsolutePath> sourcePaths, string targetPath)
+        private static void ConfigureBindFilter(IntPtr hJob, IReadOnlyCollection<ExpandedAbsolutePath> sourcePaths, string targetPath, IEnumerable<string> exclusionPaths)

```

Comment:
Just checking if the change in name is intentional...

Reply:
In this case yes, the method is BindFlt related